molgenis-metadata-manager/src/main/frontend/src/store/actions.js

```diff
@@ -32,7 +32,12 @@ export default {
     get({apiUrl: '/api'}, '/v2/sys_md_EntityType?num=10000')
       .then(response => {
         const nonSystemEntities = response.items.filter(function (item) {
-          return !item.package.id.startsWith('sys')
+          // TODO we should filter on parent package is sys, a user can also select a package name starting with sys_
```

Comment:
The id 'sys' is created by the system.
We are not checking the name/label but the ID. 

startsWith() is used because we have sys, sys_idx, sys_md etc
If you create a Package via EMX with a name that starts with sys, you will get an error telling you that one already exists.

If you create a Package via the dataexplorer, you will get a UUID